src/Components/Login/index.js

```diff
@@ -2,32 +2,67 @@ import React, { Component } from 'react';
 import './login.css';
 
 export default class Login extends Component {
+    state = { email:"@htg.nl", password:"" };
+
+    onSubmit(event) {
+        console.log("Doet ie iets?")
+        event.preventDefault();
+        localStorage.setItem( "loggedIn", true);
+    }
+
     render() {
         return (
-            <div className="login">
-                <tr>
-                    <td>
-                        E-mail:
-                    </td>
-                    <td>
-                        &nbsp;&nbsp;
-                    </td>
-                    <td>
-                        <input type="text" name="email" value="@htg.nl" />
-                    </td>
-                </tr>
-                <tr>
-                    <td>
-                        Wachtwoord:
-                    </td>
-                    <td>
-                        &nbsp;&nbsp;
-                    </td>
-                    <td>
-                        <input type="text" name="password" value="" />
-                    </td>
-                </tr>
-            </div>
+            <>
+                <div className="title">
+                    <b>Login pagina</b>
+                </div>
+                <div className="login">
+                    <table>
```

Comment:
Deze table is lelijk imo, misschien een reusable table component introduceren? Zie voorbeeld https://gist.github.com/peterschriever/3bbeb929d3e58ee2e67f3889d601dd55